utils/constants/network_ids.go

```diff
@@ -25,6 +25,10 @@ const (
 	UnitTestID uint32 = 10
 	LocalID    uint32 = 12345
 
+	// IDs used for validating mainnet and fuji in e2e
+	TestMainnetID uint32 = 808
+	TestFujiID    uint32 = 909
```

Comment:
we should probably add naming for these new networks and mapping networkIDs <-> networkName, simiarl to other networks.